help/develop/domains/chem/cheminformatics.md

```diff
@@ -18,7 +18,7 @@ or come up with your own solution.
 
 For convenience, some of the commonly used functions are exposed via the [JS API](../packages/js-api.md). Use `grok.chem`
```

Comment:
I believe, we can rewrite this sentence too. There is no need to specify For convenience and use passive voice.
My variant is:
`We expose some commonly used functions via JS API`.

Reply:
Also, it is better to specify purpose firstly.
`To invoke the methods, use grok.chem
`

Reply:
`Most of them are asynchronous due to/because they use...`. Or we can even make two separate sentences out of this huge.

Reply:
Done